assets/stylesheets/_components.scss

```diff
@@ -33,6 +33,7 @@
 @import 'components/screen-reader-text/style';
 @import 'components/segmented-control/style';
 @import 'components/spinner/style';
+@import 'components/token-field/style';
```

Comment:
This shouldn't be needed, the stylesheet for `token-field` is already included as "modular CSS" (see [here](https://github.com/Automattic/wp-calypso/blob/b5b88e6b061b790e237135ee8ac46c0dcc7dca2d/client/components/token-field/index.jsx#L21)).

Reply:
If specificity is a problem, you should add the `.wp-core-ui.wp-admin` prefix to [here](https://github.com/Automattic/woocommerce-services/blob/master/webpack.config.js#L125).

Reply:
Fixed in 9d1f95d4. Thank you!